deployments/helm-chart/values.yaml

```diff
@@ -74,6 +74,17 @@ controller:
   ## The termination grace period of the Ingress controller pod.
   terminationGracePeriodSeconds: 30
 
+  ## The resources of the Ingress controller pods.
+  resources: {}
+  # To specify resources, uncomment the following lines, adjust them as necessary,
```

Comment:
I'd prefer to remove these lines. (79-80)
I think it's reasonable to expect the user to know this and we don't have similar comments elsewhere.